### ISSUE TITLE
Crash at start of the "Estimate Cameras/Landmarks" step.

### ISSUE THREAD

**Issue by BASS10** [Kitware/TeleSculptor#492]
Hello,
Thanks for the great software. I am excited about the possibilities. Unfortunately I haven't got it to work yet.

The feature tracking step seems to run successfully but I get a crash as soon as I run the estimation step.

I have tried processing three different videos on 2 different computers. Both machines are running Windows 10 64-bit. Both with Nvidia GPUs.

I am using version 1.1.2 of TeleSculptor

I have included one of the log files in case this sheds some light on the matter. There are several "failed to triangulate" errors at the end of the log.

[telesculptor.log](https://github.com/Kitware/TeleSculptor/files/6584788/telesculptor.log)

Apologies in advance if this is an operator error!

**Comment by mleotta** [comment]
I would start by verifying that the program works for you on the sample data that is known to work.  Try downloading [09172008flight1tape3_2.mpg](https://data.kitware.com/#item/5ef11b419014a6d84ed53971) as listed on the [examples](https://github.com/Kitware/TeleSculptor/tree/master/examples) page and see if that works.

If the example data works, then this issue is likely triggered by the videos you are testing.  TeleSculptor was designed with narrow field of view focal length aerial imagery with orbital flight patterns in mind.  We are still working to make it work more reliably on more general data.  Your videos may need some configuration changes to make it work.  However, I would not expect it to crash.  We should at least resolve the crash.  If the example video works and yours does not, would you be able to share a video that crashes so that we can reproduce and debug?

**Comment by BASS10** [comment]
Thanks for the suggestion. Unfortunately I still get the crash when using the example video.

[telesculptor.log](https://github.com/Kitware/TeleSculptor/files/6590142/telesculptor.log)

**Comment by BASS10** [comment]
I tried a third computer and still the same result. Please let me know if I can do anything to help diagnose this.

Do I need the CUDA SDK or any other prerequisites?

**Comment by BASS10** [comment]
I have tried running both End-to-End and step by  step. The crash appears to happen at the same point either way; about 3 seconds after the start of the estimation step.

If I load the project after the crash it shows the previously computed track.

Let me know if you want any of the project files before/after the crash or anything else.

**Comment by BASS10** [comment]
Oh, and yes, if I try to run the estimation again on the re-loaded project, it crashes just the same.

**Comment by mleotta** [comment]
I'll need to see if I can reproduce this crash on Windows for v1.1.2.  If you could upload your `tracks.txt` and project `.conf` file that might be helpful, but I'm not sure if there is a Github limit on file attachment size.

**Comment by BASS10** [comment]
[files.zip](https://github.com/Kitware/TeleSculptor/files/6610394/files.zip)

**Comment by mleotta** [comment]
I've been able to reproduce this.  I'm not sure why we didn't see this before.  Thanks for reporting it!

I haven't had time to debug this yet, but I'll try to find some time to work on that soon.

**Comment by BASS10** [comment]
Glad to help in some small way. Thanks again for Telesculptor, it's a very promising tool, I look forward to seeing how it develops.

**Comment by mleotta** [comment]
Sorry for the delay on this.  I finally made some time to dig into this.  I was able to reproduce the crash using the installer that's available for download, but when I rebuilt that branch to debug it I am no longer able to reproduce it and I can't reproduce it in a new build of the installer using this tag.  I'm not sure what happen.  Maybe some file in the existing installer got corrupted.   It seems like the solution might be to just replace the old installer with the new one.
